crates/rpc-replay/src/transactions.rs

```diff
@@ -122,6 +123,92 @@ fn l1_handler_to_blockifier(
     Ok(blockifier::transaction::transaction_execution::Transaction::L1HandlerTransaction(l1_handler))
 }
 
+fn create_contract_address(
```

Comment:
`create` seems a bit vague here. Maybe `derive` would be better. Also, if there's any documentation you could link here, it would help

Reply:
or `calculate_contract_address_from_deploy_tx` (a bit obnoxious, but very clear...)